Bottom: be8d75f7d9372464eb70c8c89bc52f63b7a321fa
Top:    98d16480b7d93ea2391c05b32a592768b0e93cef
Author: Ian Kent <ikent@redhat.com>
Date:   2013-10-17 10:20:56 +0800

Refresh of autofs-5.0.7-fix-automount-8-mailing-list-info.patch

---

diff --git a/man/automount.8 b/man/automount.8
index bfcb41e..c1895c4 100644
--- a/man/automount.8
+++ b/man/automount.8
@@ -181,7 +181,7 @@ Don't know, I've fixed everything I know about.
 The documentation could be better.
 
 Please report other bugs along with a detailed description to
-<autofs@linux.kernel.org>. For instructions on how to join the list
-and for archives visit http://linux.kernel.org/mailman/listinfo/autofs
+<autofs@vger.kernel.org>. Please visit http://vger.kernel.org/vger-lists.html#autofs
+for information about the list.
 .SH AUTHOR
 H. Peter Anvin <hpa@transmeta.com> and Ian Kent <raven@themaw.net>.
